test/parallel/cobegin/hilde/reports.prediff

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-sort < $2 > $2.prediff.tmp && mv $2.prediff.tmp $2
+sort < $2 | grep -v "Waiting for more work" > $2.prediff.tmp && mv $2.prediff.tmp $2
```

Comment:
The test in question is for deadlock detection. I don't personally think it's important whether or not the deadlock detection printout includes an additional "Waiting for more work" but I did investigate a little bit to answer your question.

First, the additional printout only occurs with CHPL_COMM=gasnet (and deadlock detection only runs with CHPL_TASKS=fifo).

And the answer is... nothing. The problem also occurs on master. Apparently this test is not run nightly in that configuration. I just like to run GASNet+fifo testing because it handles over-subscription better.